content/ko/accessibility.html

```diff
@@ -0,0 +1,70 @@
++++
+title = "접근성 설명 "
+meta_description = "Vote.gov is committed to making its digital content accessible to individuals with disabilities by meeting or exceeding the requirements of Section 508 of the Rehabilitation Act"
+translationKey = "accessibility"
++++
+<div class="usa-prose">
+<p>vote.gov사이트는 접근성 향상을 위해 최선을 다해 노력하고 있습니다. 우리의 정책은 장애가 있는 개인들을 포함하여 누구나 모든 투표 관련 자원을 평등하게 접근할 수 있도록 보장하는 것입니다. vote.gov 웹사이트는 특정 기술에 의존하여 장애가 있는 개인들이 웹사이트의 정보를 이용하는 데 도움이 되는 도구가 원활하게 작동할 수 있도록 합니다. 이러한 기술에는 하이퍼텍스트(HTML), 와이-아리아(WAI-ARIA), 연속형 문서 양식(CSS) 및 자바스크립트(JavaScript)를 포함합니다. 이러한 기술을 통해 화면 판독기, 화면 확대기, 음성 인식 소프트웨어 및 기타 보조 기술과 함께 사이트를 쉽게 이용할 수 있습니다.</p>
+
+<p><strong>내용</strong></p>
+<ul>
+  <li><a href="#statement">접근성 안내</a></li>
+  <li><a href="#how-we-support-and-maintain">접근성 지원 및 유지 방법 안내</a></li>
+  <li><a href="#our-standards">접근성 기준 안내</a></li>
+  <li><a href="#help-feedback-and-formal-complaints">도움말, 피드백 및 불편사항 신고 안내</a></li>
+</ul>
+
+
+<h2 id="statement">접근성 안내</h2>
+<p>vote.gov 사이트 팀은 컴퓨터 하드웨어를 보고, 듣고, 작동하는 데 어려움이 있거나 인지 또는 학습에 어려움을 겪는 사람들을 포함하여 모든 유권자들에게 접근성이 용이한 웹사이트를 구현하기 최선을 다하고 있습니다. 우리는 또한 영어가 모국어가 아닌 사람들을 포함하여 소외계층에게 필요한 도움을 제공하고자 노력하고 있습니다. 접근성 향상은 지속적인 노력이며, 콘텐츠를 제작하고, 문서를 작성하고, 웹사이트를 구축하는 직원에게 정기적인 교육을 제공함으로써 사이트를 개선하고 있습니다.</p>
+
+<h2 id="how-we-support-and-maintain">접근성 지원 및 유지 방법 안내</h2>
+<p><strong>우리는 다음과 같은 방법으로 vote.gov사이트의 접근성을 보장합니다.</strong></p>
+<ul>
+  <li>수동으로 키보드 및 스크린 리더 접근성을 검사합니다.</li>
+  <li>시맨틱 섹션 헤딩을 사용하여 콘텐츠 구성합니다.</li>
+  <li>우리 웹사이트에서 제공하는 모든 링크와 대화형 부분에 키보드를 사용하여 접근할 수 있도록 허용합니다.</li>
+  <li>"메인 화면으로 바로가기" 기능을 포함합니다.</li>
+  <li>스크린 리더 사용자를 위해 외부 링크에 새 창에서 열기"를 알리는 코드를 추가합니다.</li>
+  <li>이미지, 아이콘 및 로고에 대한 자세한 대체 텍스트를 제공합니다.</li>
+  <li>의미상 문맥을 전달하고 사용자 경험을 제공하기 위해 콘텐츠 번역에 실제로 사람들이 참여합니다.</li>
+  <li>사용자가 선호하는 설정에 따라 텍스트 크기를 조정할 수 있도록 허용합니다.</li>
+  
+</ul>
+
+<h2 id="our-standards">접근성 기준 안내</h2>
+<p>우리의 웹페이지는 기술 요구사항을 명시한 연방 법 508조를 준수하여, <a href="https://www.section508.gov/">섹션508항</a> 기준에 부합하거나 이를 능가하도록 설계되었습니다. 우리는 또한 <a href="https://www.w3.org/tr/wcag21">W3C 웹사이트</a>에 기술되어 있는 W3C 웹 콘텐츠 접근성 지침(WCAG) 2.1을 준수합니다. 우리는 AA수준 표준을 충족하며, 다시 말해, 대부분의 사람들이 대부분의 상황에서 콘텐츠에 접근이 가능합니다.</p>
```

Comment:
We need to add ` {{< inenglish >}}` inside the link tags for untranslated links.